setup.cfg

```diff
@@ -0,0 +1,15 @@
+[flake8]
+# References:
+# http://flake8.readthedocs.org/en/latest/config.html
+# http://flake8.readthedocs.org/en/latest/warnings.html#error-codes
+#
+# Style checks turned on:
+#   F - all pyflake errors
+#   E101 - indentation contains mixed spaces and tabs
+#   E111 - indentation is not a multiple of four
+#   E501 - line too long (see max-line-length)
+
+exclude = bokeh/models/__init__.py, bokeh/models/widgets/__init__.py
+ignore = E,W
+select = F,E101,E111,E501
+max-line-length = 120
```

Comment:
@bryevdv what's your opinion about imposing a max line length? We can always change/remove it of course, so maybe we can try imposing a relatively generous width to pick out the worst cases?